content/ai/concepts/chatgpt/chatgpt.md

```diff
@@ -0,0 +1,96 @@
+---
+Title: 'ChatGPT'
+Description: 'ChatGPT is a publically available web-based chatbot and API interface created by OpenAI. It connects to a large language model (LLM) called GPT (Generative Pre-Trained Transformer). GPT includes some of the largest models ever created. After the model training, there is further fine-tuning to improve its generated responses.'
+Subjects:
+  - 'Machine Learning'
+  - 'Data Science'
+  - 'Large Language Model'
+  - 'Transformers'
+Tags:
+  - 'AI'
+  - 'Transformers'
+  - 'Embeddings'
+  - 'Prompts'
+  - 'Prompt Engineering'
+  - 'Attention'
+CatalogContent:
+  - 'machine-learning'
+  - 'paths/data-science'
+---
+
+`ChatGPT` is a publically available web-based chatbot and API interface created by `OpenAI`. It connects to a large language model (LLM) called GPT (Generative Pre-Trained Transformer). GPT includes some of the largest models ever created. After the model training, there is further fine-tuning to improve its generated responses.
+
+## How ChatGPT Works
```

Comment:
Interestingly, we don't mention (and ChatGPT didn't suggest to us) to talk about Transformers or LLMs. We talk about it being a GAN (Generative Adversarial Networks) and will have a whole lesson on how those work.

Reply:
I don't think ChatGPT is a GAN. If it stands for generative pre-trained transformer - we should be talking about transformers no?